spec/order_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'order'
+require 'dish'
+require 'pry'
+
+describe Order do     #less let
+
+  subject(:order) { described_class.new(menu) }
+  let(:takeaway) { Takeaway.new(menu: menu, order: order, sms: sms, config: :config) }
+  let(:menu) { double :menu, dishes: [dish, dish2] }
+  let(:dish) { Dish.new(name: "Pizza", price: 7.50) }
+  let(:dish2) { Dish.new(name: "Soup", price: 3.20) }
+  let(:dish1) { double :dish }
+  let(:sms) { instance_double "SMS", deliver: nil }
```

Comment:
Mixing real objects with doubles. The only real object in this spec file should be from the Order class.